Source/Collection/Plugins/PluginAction/CollectionScrollViewDelegateProxyPlugin.swift

```diff
@@ -71,3 +71,48 @@ public extension BaseCollectionPlugin {
     }
 
 }
+
+/// Proxy of  `CollectionScrollViewDelegateProxyPlugin` events
+#if os(iOS)
+@available(iOS 13.0, *)
+public extension CollectionScrollViewDelegateProxyPlugin {
+
+    // MARK: - Private static properties
+
+    private static var didScrollCompositionLayoutSection = BaseEvent<ItemsInvalidationResult>()
+
+    // MARK: Nested types
+
+    typealias ItemsInvalidationResult = (items: [NSCollectionLayoutVisibleItem], offset: CGPoint, environment: NSCollectionLayoutEnvironment)
+
+    // MARK: - Ppublic properties
+
+    /// Setter requires calling the NSCollectionLayoutSection method `setHorizontalScroll(type: with plugin:)`
```

Comment:
```suggestion
    /// - Note: To enable events you should link your `NSCollectionLayoutSection` with `CollectionScrollViewDelegateProxyPlugin` using method `NSCollectionLayoutSection.setHorizontalScroll`
```

Кстати вопрос. 
Если у нас несколько горизонтальных секций которым надо навесить это поведение, надо будет 

- один раз создать плагин
- каждой секции вызвать `setHorizontalScroll`
- как-то различать секции в `handleVisibleItemsInvalidation`
?

Нам достаточно будет информации из `ItemInvalidationResult` чтобы при необходимости сделать разные трансформации каждой секции?

Reply:
Информации будет достаточно ведь там есть indexPath